tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -5,21 +5,22 @@ export class God {
 }
 
 export class Human {
-  constructor(name) {
+  constructor( name, sex ) {
     this.name = name;
+    this.sex = sex;
   }
 }
 
 export class Man extends Human {
   constructor( name, sex ) {
-    super(name);
+    super(name, sex);
```

Comment:
it would be better if you will just pass `super(name, 'male')` as your object have property which have to have parent

Reply:
Done